### DISCUSSION TITLE
asp .net core example for razor pages and API

### DISCUSSION THREAD

**Discussion by vitalybrandes** [logto-io/logto#5108] [Q&A] [answered]
Is there any asp .net core example for razor pages and API.
I trying to get the permissions list to validate API , but unfortunately i got only roles.
Also, that i the way to get received AccessToken?

**Comment by simeng-li** [comment, marked as answer]
https://github.com/logto-io/csharp/blob/master/src/Logto.AspNetCore.Authentication/docs/tutorial.md#api-resources

@vitalybrandes  Hope this might help.

**Reply by vitalybrandes** [reply]
Perfectly. thanks!

**Comment by vitalybrandes** [comment]
Case still open....
I am not able to get permissions for.multiple resources in .net core. It's got only one single resource....
